.github/workflows/release.yml

```
@@ -49,85 +345,80 @@ jobs:
+          merge_mac_yml latest-mac.yml
+          merge_mac_yml beta-mac.yml
+
+          if [ -z "$(ls -A release_assets)" ]; then
+            echo "Error: No release assets found"
+            exit 1
           fi
 
           ls -la release_assets/
 
       - name: Create Draft Release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)